packages/contracts/src/contracts/current/protocol/Exchange/LibOrder.sol

```diff
@@ -48,6 +50,8 @@ contract LibOrder {
         uint256 takerFeeAmount;
         uint256 expirationTimeSeconds;
         uint256 salt;
+        uint8 makerAssetProxyId;
```

Comment:
@abandeali1 - I added these new fields to the end of the struct so it would be easier to extend functions that access fields in assembly, like in `MixinWrapperFunctions.sol`. Although I can reorder so they're higher, closer to the maker/taker token addresses if that would make more sense.